meal-planner/src/App.js

```diff
@@ -1,59 +1,74 @@
 import { useState } from 'react'
 import styled from 'styled-components/macro'
 import woodBackground from './assets/wood_background.jpg'
-import logo from './assets/logo.png'
+import logo from './assets/logo.svg'
 import Button from './components/Button'
-import MealSelect from './components/MealSelect'
+import MealSelectMenu from './components/MealSelectMenu'
+import { v4 as uuidv4 } from 'uuid'
 
 export default function App() {
-    const weekdays = [
-        'Montag',
-        'Dienstag',
-        'Mittwoch',
-        'Donnerstag',
-        'Freitag',
-        'Samstag',
-        'Sonntag',
-    ]
+    const [weekdays, setWeekdays] = useState([
+        { weekday: 'Montag', id: uuidv4(), meal: '' },
+        { weekday: 'Dienstag', id: uuidv4(), meal: '' },
+        { weekday: 'Mittwoch', id: uuidv4(), meal: '' },
+        { weekday: 'Donnerstag', id: uuidv4(), meal: '' },
+        { weekday: 'Freitag', id: uuidv4(), meal: '' },
+        { weekday: 'Samstag', id: uuidv4(), meal: '' },
+        { weekday: 'Sonntag', id: uuidv4(), meal: '' },
+    ])
 
-    const [isWeekday, setIsWeekday] = useState()
+    const [selectedWeekday, setSelectedWeekday] = useState(null)
 
     function handleClick(event) {
-        setIsWeekday(event.currentTarget.id)
+        setSelectedWeekday(event.currentTarget.id)
+    }
+
+    function selectMeal(meal, id) {
+        setSelectedWeekday(null)
```

Comment:
Maybe use empty strings in stead of ``null`` to make it easier to grasp.

Reply:
I agree :)

Reply:
That's because null is supposed to be "the intentional undefined" so That everybody Knows ist hasn't been forgotton. But Right, Could also be an empty string.